Sources/sentry/include/SentryApp.h

```diff
@@ -5,4 +5,28 @@
 
 #include <sentry.h>
 
-#endif
\ No newline at end of file
+#ifdef _WIN32
+#include <atomic>
+
+// The type of swift::swift_getFatalErrorMessageBuffer
+using swift_getFatalErrorMessageBuffer_t = std::atomic<const char *> *(*)();
+
+inline void *getFatalErrorMessageHandle() {
+  HMODULE hSwiftCore = GetModuleHandleA("swiftCore.dll");
```

Comment:
This will leak a handle. It might be what we want/need here, but we should document this in a comment.

Reply:
Done

Reply:
It could still use that comment :)